components/brex/actions/list-transactions-for-selected-cash-account/common.mjs

```diff
@@ -0,0 +1,42 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  async run ({ $ }) {
+    const {
+      max,
+      postedAtStart,
+      cashAccount,
+    } = this;
+
+    const DEFAULT_LIMIT = 100;
+    const limit = max
+      ? Math.min(DEFAULT_LIMIT, parseInt(max))
+      : DEFAULT_LIMIT;
+
+    let items = [];
+    let cursor;
+    do {
+      const res = await axios($, this.brexApp._getAxiosParams({
+        method: "GET",
+        path: `/v2/transactions/cash/${cashAccount}`,
+        params: {
+          limit,
+          posted_at_start: postedAtStart,
+          cursor,
+        },
+      }));
+
+      if (res.items) {
+        items = [
+          ...items,
+          ...res.items,
+        ];
+      }
+
+      cursor = res.next_cursor;
+    } while (items.length < max && cursor);
+
+    $.export("$summary", "Transactions successfully fetched");
```

Comment:
can you add a count of how many transactions we fetched?
"Successfully fetched 35 transactions" for example

Reply:
Okay :)